aws/client/default_retryer.go

```diff
@@ -32,7 +31,13 @@ func (d DefaultRetryer) MaxRetries() int {
 
 // RetryRules returns the delay duration before retrying this request again
 func (d DefaultRetryer) RetryRules(r *request.Request) time.Duration {
-	delay := int(math.Pow(2, float64(r.RetryCount))) * (rand.Intn(30) + 30)
+	// Set the upper limit of retrying at ~five minutes
```

Comment:
Should clarify this is the upper limit on retry delay not number of retry attempts.